exercism/diamond/diamond.js

```diff
@@ -0,0 +1,44 @@
+var Diamond = function() {}
+
+Diamond.prototype.makeDiamond = function(letter) {
+  if (letter === 'A') {
+    return 'A\n'
+  }
+  var alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split('')
+  var character = ' '
+  var middle = alphabet.indexOf(letter) + 1
```

Comment:
middleOfDiamond would be a better name for this.
Try and make your variables as meaningful as possible.
It took me a while to figure out what you meant by middle.